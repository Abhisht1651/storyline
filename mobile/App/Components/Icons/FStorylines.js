import React from 'react'
import { View, Text } from 'react-native'

// Styles
import styles from '../Styles/IconStyles/FStorylinesStyles'
import FollowedStorylines from '../../Images/Icons/icon-followedstorylines.svg'

const FStorylines = () => {
  return (
    <View style={styles.cardBox}>
      <View style={styles.profileIconBox}>
        <FollowedStorylines />
      </View>
      <View style={styles.TextBox}>
        <Text style={styles.TextStyles}>Followed Storylines</Text>
      </View>
    </View>
  )
}

export default FStorylines
