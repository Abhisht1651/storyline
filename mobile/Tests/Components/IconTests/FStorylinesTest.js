import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'

import FStorylines from '../../../App/Components/Icons/FStorylines'

test('FStorylines Icon component renders correctly', () => {
  const tree = renderer.create(<FStorylines />).toJSON()
  expect(tree).toMatchSnapshot()
})
