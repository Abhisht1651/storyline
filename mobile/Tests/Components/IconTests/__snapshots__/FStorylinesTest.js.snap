// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FStorylines Icon component renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "transparent",
      "flexDirection": "row",
      "height": 27,
      "opacity": 1,
      "width": 198,
    }
  }
>
  <View
    style={
      Object {
        "height": 27,
        "width": 24,
      }
    }
  >
    <SvgMock />
  </View>
  <View
    style={
      Object {
        "height": 20,
        "marginLeft": 10,
        "marginTop": 4,
        "width": 165,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#1F1F60",
          "fontFamily": "BananaGrotesk-Medium",
          "fontSize": 18,
          "letterSpacing": -0.12,
          "opacity": 1,
          "textAlign": "left",
        }
      }
    >
      Followed Storylines
    </Text>
  </View>
</View>
`;
